ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/MergeDocs.groovy

```diff
@@ -0,0 +1,32 @@
+package com.marklogic.gradle.task
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.gradle.api.GradleException
+import org.gradle.api.tasks.TaskAction
+
+class MergeDocs extends HubTask  {
```

Comment:
This should be MergeDocsTask for consistency with other tasks. 

I'm wondering too if we can be more specific and call it MergeEntitiesTask. Is it restricted to entities, or does smart mastering not depend on having entities defined?

Reply:
Changed to be consistent with other tasks and agree that "entities" is more accurate. Class name is now MergeEntitiesTask and the call is now hubMergeEntities